docs/modules/item.md

```diff
@@ -35,7 +35,20 @@ and it would involve removing a current enchantment, add the flag `-o`. If you a
 enchantment level onto an item which is not normally allowed (often referred to as "unsafe"), add the flag `-u`, though
 this requires an extra permission (`nucleus.enchant.unsafe`).
 
+## Repair
+
+Nucleus can be used to repair items in a player's inventory using `/repair`. By default, `/repair` will only target the item held in a player's main hand. By using and combining the availible flag arguments, specific inventory locations or the entire inventory can be repaired.
+
+### Flag Arguments
+
+* `/repair -m` repairs the player's main hand (default behavior).
```

Comment:
If we're going all `en_GB` (and let's face it, we all know that's the _correct_ way to spell!), you're missing a "u" in "behaviour" ;)

In all honesty, I'd not thought about the UK/US differences when writing these so I can't say I'm horribly bothered, but I guess we should be consistent.

Reply:
I personally was going for consistency.